devel/idevicerestore/files/patch-postrelease-fixes.diff

```diff
@@ -0,0 +1,86 @@
+diff --git configure.ac configure.ac
+index 1db709a..06bb4ed 100644
+--- configure.ac
++++ configure.ac
+@@ -53,8 +53,8 @@ case ${host_os} in
+   *mingw32*|*cygwin*)
+     AC_MSG_RESULT([yes])
+     win32=true
+-    GLOBAL_CFLAGS+="-DWIN32 -D__LITTLE_ENDIAN__=1"
+-    AC_LDFLAGS+="-static-libgcc"
++    GLOBAL_CFLAGS+=" -DWIN32 -D__LITTLE_ENDIAN__=1"
++    AC_LDFLAGS+=" -static-libgcc"
```

Comment:
Surely this part of the patch is irrelevant to us since it's only for mingw32 and cygwin.